### DISCUSSION TITLE
Welcome to NimBLE-Arduino Discussions!

### DISCUSSION THREAD

**Discussion by h2zero** [h2zero/NimBLE-Arduino#334] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by OscarMuRu99** [comment]
I have a problem, I am connecting 2 clients to a server using 3 esp32s, when I want to know the rssi using ->getRssi() it doesn´t show the value, it resets the system continuously. I don´t know if it has happened before, is there any way to fix this? I want to try and make a room presence type of project using the rssi value. I am using the Nimble_client example code and the Nimble_server example code. I would appreciate any kind of response. Thanks in advance.
